docs/virtual_cap_pools.md

```diff
@@ -2,135 +2,138 @@
+school_a.over_order_reclaimed_laptops #=> 0
+school_a.raw_laptops_ordered  #=> 0
+```
 
-`ResponsibleBody` _must_
+that means this school was allocated 10 laptops, 10 (10-0-0) is the maximum number that could be ordered (raw cap) and it has ordered 0 laptops so far:
+```
+school_a.raw_laptops #=> [10, 10, 0] # raw_allocation: 10, raw_cap: 10, devices_ordered: 0 for laptops
+```
 
-* Have `vcap` set to `true`
+School B, in the same pool, has these numbers:
+```
+school_b.order_state #=> 'can_order_for_specific_circumstances'
+school_b.raw_laptop_allocation #=> 10
+school_b.circumstances_laptops #=> -3
+school_b.over_order_reclaimed_laptops #=> 0
+school_b.raw_laptops_ordered #=> 0
+```
+so 
+```
+school_b.raw_laptops #=> [10, 7, 0]
+```
 
-`School` _must_
+Similarly, a third school in the pool, School C:
+```
+school_c.order_state                  #=> 'cannot_order'
+school_c.raw_laptop_allocation        #=> 10
+school_c.circumstances_laptops        #=> 0
+school_c.over_order_reclaimed_laptops #=> 0
+school_c.raw_laptops_ordered          #=> 0
+```
+so
+```
+school_c.raw_laptops #=> [10, 0, 0] # raw_cap is 0 instead of (10-0-0) because the school is set to can't order devices.
+```
 
-* Be centrally managed - `school.orders_managed_centrally?` returns `true`
-* Belong to the same responsible body as the `VirtualCapPool`
-* Have at least 1 device allocation record
-* Not already be in the `VirtualCapPool`
+Therefore, if these are all the schools in the vcap, we can say:
 
-#### Automatically adding schools
+- vcap allocation equals to 30 laptops
+- vcap cap equals to 20 laptops (maximum number of laptops allowed to be ordered)
+- vcap laptops ordered equals to 0 (laptops ordered so far by all the schools in the pool)
 
-Schools are normally automatically added to a virtual pool when they are enabled for ordering and the above prerequisites are in place.  This takes place via the `UpdateSchoolDevicesService#update!` method used to change the school's order state.
+When the vcap gets computed, their 3 allocation number fields gets populated like this:
+```
+vcap.laptop_allocation: 30
+vcap.laptop_cap: 20
+vcap.laptops_ordered: 0
+```
+We can get these numbers listed together with the #laptops method:
+```
+vcap.laptops #=> [30, 20, 0]
+```
 
-#### Manually add a school
 
-Sometimes it may be necessary to add schools to a pool outside of this process.
+## Placing devices orders
+[Computacenter outgoing api](./computacenter_outgoing_api.md) document describes how GHwT system reports individual school and vcap allocation numbers to the supplier (Computacenter) for the users to place device orders.
 
-1. Ensure the prerequisites above are in place
-2. Add the school to the responsible body's virtual pools
+When a user places an order in the suppliers portal (CC's TechSource website) it will get reported instantaneously back to GHwT so that individual school and vcap allocation numbers get inmediately updated accordingly.
```

Comment:
```suggestion
When a user places an order in the supplier's portal (CC's TechSource website) it will get reported instantaneously back to GHwT so that individual school and vcap allocation numbers get inmediately updated accordingly.
```